### ISSUE TITLE
CI fails sporadically

### ISSUE THREAD

**Issue by daffl** [donejs/donejs#546]
Today I merged a couple of documentation changes and the builds keep failing sporadically in different Node versions. See https://travis-ci.org/donejs/donejs/builds

@matthewp any ideas why? Looks like it just stops and times out.

**Comment by matthewp** [comment]
Looking into it.

**Comment by matthewp** [comment]
Made an improvement by upgrading the Firefox version in Travis. But let's keep open for now.

**Comment by daffl** [comment]
After #622 this seems to be solved. I created #623 as a follow-up issue for that.
